docs/de/developer-portal-page.md

```diff
@@ -20,47 +20,75 @@ Zertifikat bezeichnet. Für PSD2-Zwecke wird das Zertfikat anhand des QcStatemen
 Nach der Einbindung des QWAC in einem XS2A Request werdem Rolle und Signatur an einem zentralen Reverse Proxy validiert, bevor die Anfrage an die XS2A-Schnittstelle weitergeleitet wird, wo sich die Bankenlogik abspielt. Unsere Dokumentation finden Sie hier: [XS2A Swagger UI](/swagger-ui.html).
 
 Die beschriebenen Komponenten und das Zusammenspiel dieser, werden in Abbildung 1.1 dargestellt:
-displayed in Figure 1.1:
 
 ![PSD2 Accelerator](assets/accelerator.svg 'Figure 1.1: Components of the PSD2 Accelerator')
 _Figure 1.1: Components of the PSD2 Accelerator_
 
-### Aktuelle XS2A Konfiguration (Bank Profil)
+### Aktive XS2A Konfiguration (Bank Profil)
+
+- SCA-Approach: Redirect
```

Comment:
Complete List of active configuration is not visible in the rendered view

Reply:
![image](https://user-images.githubusercontent.com/191864/53747529-bfcd3980-3ea3-11e9-934c-c4c3e7d78472.png)

Looks correct, doesn't it?

Reply:
Cant reproduce it again. It works